server.js

```diff
@@ -1,60 +1,60 @@
-// TASK 1: Setting up server
+// // TASK 1: Setting up server
 
 
-const http = require('http');
-
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
+// const http = require('http');
 
-const PORT = 80;
-const HOSTNAME = '0.0.0.0';
+// const server = http.createServer((req, res) => {
+//   res.statusCode = 200;
+//   res.setHeader('Content-Type', 'text/plain');
+//   res.end('Hello World\n');
+// });
 
-server.listen(PORT, HOSTNAME, () => {
-  console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
-});
+// const PORT = 80;
+// const HOSTNAME = '0.0.0.0';
 
+// server.listen(PORT, HOSTNAME, () => {
+//   console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
+// });
 
 
 
-// Task 2 : Connecting with mongo db data base 
 
+// // Task 2 : Connecting with mongo db data base 
 
-const http = require('http');
-const { MongoClient } = require('mongodb');
 
-const PORT = 80;
-const HOSTNAME = '0.0.0.0';
+// const http = require('http');
```

Comment:
The uri variable seems to be redundant and not used in the code. It should be removed.